cluster/external-worker.yml

```diff
@@ -0,0 +1,59 @@
+name: ((deployment_name))
+
+releases:
+- name: concourse
+  version: ((concourse_version))
+  sha1: ((concourse_sha1))
+  url: https://bosh.io/d/github.com/concourse/concourse?v=((concourse_version))
+- name: garden-runc
+  version: ((garden_runc_version))
+  sha1: ((garden_runc_sha1))
+  url: https://bosh.io/d/github.com/cloudfoundry/garden-runc-release?v=((garden_runc_version))
+
+stemcells:
+- alias: trusty
+  os: ubuntu-trusty
+  version: latest
+
+instance_groups:
+- name: worker
+  instances: ((instances))
+  azs: ((azs))
+  vm_type: ((worker_vm_type))
+  stemcell: trusty
+  networks: [{name: ((network_name))}]
+  jobs:
+  - name: groundcrew
+    release: concourse
+    consumes: {baggageclaim: {from: worker-baggageclaim}}
+    properties:
+      tags: ((worker_tags))
+      tsa:
+        host: ((tsa_host))
+        port: 2222
+        host_public_key: ((tsa_host_key.public_key))
+        worker_key: ((worker_key))
+      garden:
+        forward_address: 127.0.0.1:7777
+      baggageclaim:
+        forward_address: 127.0.0.1:7788
+  - name: baggageclaim
+    release: concourse
+    provides: {baggageclaim: {as: worker-baggageclaim}}
+    properties:
+      log_level: debug
+      bind_ip: 127.0.0.1
+  - name: garden
+    release: garden-runc
+    properties:
+      garden:
+        forward_address: 127.0.0.1:7777
+        listen_network: tcp
+        listen_address: 0.0.0.0:7777
```

Comment:
I would lock this down to `127.0.0.2:7777`, since it has to be forwarded anyway. Depending on where it's deployed, a private network may not be an option.

Reply:
What would be the implications, can you elaborate a little on this? Feel free to change it.

Reply:
Ok, got it. Done.